sdk/core/azure-core/src/http/http.cpp

```diff
@@ -153,3 +155,13 @@ void Azure::Core::Http::Details::InsertHeaderWithValidation(
   // insert (override if duplicated)
   headers[headerName] = headerValue;
 }
+
+std::string Azure::Core::Http::Internal::RangeToString(Range range)
+{
+  std::string rangeString = "bytes=" + std::to_string(range.Offset) + "-";
```

Comment:
nit: use a named `unit` constant like this:
https://github.com/Azure/azure-sdk-for-net/blob/8b100f13b70aa00fdb69304500ea7e2a6a142322/sdk/core/Azure.Core/src/HttpRange.cs#L17-L18

Reply:
I don't think the format is correct.

it's actually [startOffset, startOffset + length - 1]

Reply:
Won't provide the toString method